From 7a0cdd86cd4f0f95f28ab9c265020ff058063102 Mon Sep 17 00:00:00 2001
Message-ID: <7a0cdd86cd4f0f95f28ab9c265020ff058063102.1778085737.git.pav@iki.fi>
In-Reply-To: <824db59fa2d845a7bcf0a00a279419c8433e0e61.1778085737.git.pav@iki.fi>
References: <824db59fa2d845a7bcf0a00a279419c8433e0e61.1778085737.git.pav@iki.fi>
From: Pauli Virtanen <pav@iki.fi>
Date: Sat, 21 Feb 2026 18:50:23 +0200
Subject: [PATCH BlueZ 08/10] doc: enable CONFIG_VIRTIO_CONSOLE in tester
 config

Enable kernel option that allows using -device virtserialport in qemu.
This is easier to make work reliably than pci-serial channel.
---
 doc/ci.config       | 1 +
 doc/test-runner.rst | 1 +
 doc/tester.config   | 1 +
 3 files changed, 3 insertions(+)

diff --git a/doc/ci.config b/doc/ci.config
index 31e49ba96..a48c1af9d 100644
--- a/doc/ci.config
+++ b/doc/ci.config
@@ -6,6 +6,7 @@
 
 CONFIG_VIRTIO=y
 CONFIG_VIRTIO_PCI=y
+CONFIG_VIRTIO_CONSOLE=y
 
 CONFIG_NET=y
 CONFIG_INET=y
diff --git a/doc/test-runner.rst b/doc/test-runner.rst
index 64715e2e7..d030787a4 100644
--- a/doc/test-runner.rst
+++ b/doc/test-runner.rst
@@ -45,6 +45,7 @@ option (like the Bluetooth subsystem) can be enabled on top of this.
 
 	CONFIG_VIRTIO=y
 	CONFIG_VIRTIO_PCI=y
+	CONFIG_VIRTIO_CONSOLE=y
 
 	CONFIG_NET=y
 	CONFIG_INET=y
diff --git a/doc/tester.config b/doc/tester.config
index 4ee306405..015e7cc1a 100644
--- a/doc/tester.config
+++ b/doc/tester.config
@@ -1,6 +1,7 @@
 CONFIG_PCI=y
 CONFIG_VIRTIO=y
 CONFIG_VIRTIO_PCI=y
+CONFIG_VIRTIO_CONSOLE=y
 
 CONFIG_NET=y
 CONFIG_INET=y
-- 
2.54.0

